docker-compose.dev.yml

```diff
@@ -0,0 +1,10 @@
+version: "3.3"
+services:
+  api:
+    build:
+      context: .
+      dockerfile: Dockerfile-dev
```

Comment:
We're not going to use the `dev` approach for docker anymore. Can you check and change the names of the Dockerfile and the docker-compose files? 🙂